recipe/migrations/cuda110.yaml

```diff
@@ -31,14 +31,22 @@ __migrator:
       - quay.io/condaforge/linux-anvil-cos7-cuda:10.1    # [linux64 and os.environ.get("BUILD_PLATFORM") == "linux-64"]
       - quay.io/condaforge/linux-anvil-cuda:10.2    # [linux64 and os.environ.get("BUILD_PLATFORM") == "linux-64"]
       - quay.io/condaforge/linux-anvil-cos7-cuda:10.2    # [linux64 and os.environ.get("BUILD_PLATFORM") == "linux-64"]
+  - quay.io/condaforge/linux-anvil-ppc64le-cuda:10.2  # [linux and ppc64le and os.environ.get("DEFAULT_LINUX_VERSION", "cos6") == "cos7"]
+  - quay.io/condaforge/linux-anvil-aarch64-cuda:10.2  # [linux and aarch64 and os.environ.get("DEFAULT_LINUX_VERSION", "cos6") == "cos7"]
       - quay.io/condaforge/linux-anvil-cuda:11.0    # [linux64 and os.environ.get("BUILD_PLATFORM") == "linux-64"]
+      - quay.io/condaforge/linux-anvil-ppc64le-cuda:11.0  # [linux and ppc64le and os.environ.get("DEFAULT_LINUX_VERSION", "cos6") == "cos7"]
+      - quay.io/condaforge/linux-anvil-aarch64-cuda:11.0  # [linux and aarch64 and os.environ.get("DEFAULT_LINUX_VERSION", "cos6") == "cos7"]
       - quay.io/condaforge/linux-anvil-cuda:11.1    # [linux64 and os.environ.get("BUILD_PLATFORM") == "linux-64"]
+      - quay.io/condaforge/linux-anvil-ppc64le-cuda:11.1  # [linux and ppc64le and os.environ.get("DEFAULT_LINUX_VERSION", "cos6") == "cos7"]
+      - quay.io/condaforge/linux-anvil-aarch64-cuda:11.1  # [linux and aarch64 and os.environ.get("DEFAULT_LINUX_VERSION", "cos6") == "cos7"]
       - quay.io/condaforge/linux-anvil-cuda:11.2    # [linux64 and os.environ.get("BUILD_PLATFORM") == "linux-64"]
+      - quay.io/condaforge/linux-anvil-ppc64le-cuda:11.2  # [linux and ppc64le and os.environ.get("DEFAULT_LINUX_VERSION", "cos6") == "cos7"]
+      - quay.io/condaforge/linux-anvil-aarch64-cuda:11.2  # [linux and aarch64 and os.environ.get("DEFAULT_LINUX_VERSION", "cos6") == "cos7"]
```

Comment:
John, could you remind me why we needed CUDA 11.1+ in this file? Don't we also have another migrator `cuda111_112.yaml`?

Reply:
Think this was needed so we could add CentOS 7 support ( https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/1232 )

Reply:
I think CUDA for `aarch64` does not support CentOS 7
https://docs.nvidia.com/cuda/archive/11.0/cuda-installation-guide-linux/index.html

Reply:
Yeah that's unrelated. Wouldn't worry about it 🙂